packages/crypto/src/networks/devnet/milestones.json

```diff
@@ -51,5 +51,11 @@
     {
         "height": 1750000,
         "vendorFieldLength": 255
+    },
+    {
+        "height": 100,
```

Comment:
Why 100?

Reply:
The final number should of course be somewhere in the future.

I was surprised to see that some unit tests use devnet config, even though we have unitnet. This seems wrong to me and I am trying to see what tests do that - they will break due to this change because they use fixtures with short block id which will be rendered invalid.

Reply:
Why would they become invalid? Old and new fixtures need to pass with the switch as the old IDs still exist.

Reply:
The tests that use devnet config break here because they use height>100 and this config denotes all blocks with height>100 should have full sha256 ids. This is temporary just to find the tests that use devnet config and fix them to use unitnet.